package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "phph-site",
+  "version": "1.0.0",
+  "description": "PHP Hampshire website",
+  "license": "MIT",
+  "devDependencies": {
+    "gulp": "^3.9.1",
+    "gulp-clean-css": "^2.3.2",
+    "gulp-concat": "^2.6.1",
+    "gulp-if": "^2.0.2",
+    "gulp-sass": "^3.0.0",
+    "gulp-uglify": "^2.0.0"
+  }
```

Comment:
Could we have a couple of scripts so we can run `npm run build` instead of `./node_modules/.bin/gulp build`?

```json
"scripts": {
    "build": "gulp build",
    "watch": "gulp"
}
```